options/locale/locale_en-US.ini

```diff
@@ -2311,9 +2313,9 @@ release.target = Target
 release.tag_helper = Choose an existing tag or create a new tag.
 release.tag_helper_new = New tag. This tag will be created from the target.
 release.tag_helper_existing = Existing tag.
-release.title = Title
+release.title = Release Title
 release.title_empty = Title cannot be empty.
-release.content = Content
+release.message = Describe this Release
```

Comment:
Suddenly title case?

Reply:
Yeah it's kind of bad habit from me, undone.